frontend/src/components/resources/resource_show_container.jsx

```diff
@@ -4,14 +4,17 @@ import { fetchResource } from '../../actions/resource_actions';
 import { fetchHabits } from '../../actions/habit_actions';
 
 
-const mapStateToProps = ({entities: { resources: { all: resources },
+const mapStateToProps = ({
+                            session: { user: currentUser },
+                            entities: { resources: { all: resources },
                                       habits: { all: habits } } },
                          {match: {params: { resourceId } } }) => {
   let resource = resources[resourceId];
   return {
     resourceId,
     resource,
     habit: resource ? habits[resource.habit] : undefined,
+    currentUser: currentUser,
```

Comment:
`currentUser,`
is syntactic sugar for the same thing as
`currentUser: currentUser,`

Reply:
you did it both ways, one in the habit show container and one in the resource show container, so like, you already know that though, lol